### PR TITLE
Prepare 4.3.0

### PR DESCRIPTION
**Pull request by radrow**

### CODE REVIEW

**Comment by radrow** [comment]
Should I just copy the exe there? Maybe there should be a post-hook for escriptizer

**Comment by radrow** [comment]
Should I just copy the exe there? Maybe there should be a post-hook for escriptizer

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,3 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix bug with standard library includes.
 ### Removed
 
+[Unreleased]: https://github.com/aeternity/aesophia_http/compare/v4.3.0...HEAD
+[4.2.1]: https://github.com/aeternity/aesophia_http/compare/v4.2.0...v4.3.0
```
Comment (tolbrino) [inline CHANGELOG.md:35]: copy paste errors?

Comment (radrow) [reply CHANGELOG.md:35]: Indeed, good catch

Comment (tolbrino) [reply CHANGELOG.md]: Still referring to aesophia_http.

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,3 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix bug with standard library includes.
 ### Removed
 
+[Unreleased]: https://github.com/aeternity/aesophia_cli/compare/v4.3.0...HEAD
+[4.2.1]: https://github.com/aeternity/aesophia_cli/compare/v4.2.0...v4.3.0
```
Comment (tolbrino) [inline CHANGELOG.md:35]: still the range is wrong

Comment (radrow) [reply CHANGELOG.md:35]: aaaaaaaaaaa

Comment (hanssv) [reply CHANGELOG.md:35]: No the version is wrong, there is no 4.2.1 for the cli?

Comment (radrow) [reply CHANGELOG.md]: Should be good now, checked the links

**Pull request merged**